app/scripts/race.js

```diff
@@ -1,95 +1,118 @@
-                    <Typography className="smallMargin" variant="title">
-                        {title}
-                    </Typography>
-                    <Typography>
-                        {text}
-                    </Typography>
-                </CardContent>
-            </Card>
-        );
-    },
+  createCard: function (title, text) {
+    return (
+      <Card className="racecard wellSpaced" raised={true}>
+        <CardContent>
+          <Typography className="smallMargin" variant="title">
+            {title}
+          </Typography>
+          <Typography>{text}</Typography>
+        </CardContent>
+      </Card>
+    );
+  },
 
-    render: function () {
-        var Header = (<div id="header" > <h1 > {this.state.name} </h1> <h2>{this.state.meet} {this.state.event} </h2></div>);
-        var Thoughts = this.createCard("General Thoughts", this.state.thoughts);
-        var TurnPoint = this.createCard("Turning Points", this.state.turnpoint);
-        var Positives = this.createCard("Positives", this.state.positives);
-        var Goal = this.createCard("Goal", this.state.goal);
-        var AE = this.createCard("Attitude: " + this.state.attitude + "\n" + "Effort: " + this.state.effort, "")
-        var SubmitDate = this.createCard("Date Submitted: ", this.state.date)
-        return (
-            <div>
-                <div className="container">
-                    <span >
-                        <button className="navButton" onClick={() => this.navigate("pastraces")}>Back</button>
-                        <button className="navButton" onClick={this.editRace}>Edit</button>
-                    </span>
-                    {Header}
-                    <div className="row">
-                        <div className="column">
-                            {Thoughts}
-                            {TurnPoint}
-                            {SubmitDate}
-                        </div>
-                        <div className="column">
-                            {Positives}
-                            {Goal}
-                            {AE}
-                        </div>
-                    </div>
-                </div>
+  render: function () {
+    var Header = (
+      <div id="header">
+        {" "}
+        <h1> {this.state.name} </h1>{" "}
+        <h2>
+          {this.state.meet} {this.state.event}{" "}
+        </h2>
+      </div>
+    );
+    var Thoughts = this.createCard("General Thoughts", this.state.thoughts);
+    var TurnPoint = this.createCard("Turning Points", this.state.turnpoint);
+    var Positives = this.createCard("Positives", this.state.positives);
+    var Goal = this.createCard("Goal", this.state.goal);
+    var AE = this.createCard(
+      "Attitude: " +
+        this.state.attitude +
+        "\n" +
+        "Effort: " +
+        this.state.effort,
+      ""
+    );
+    var SubmitDate = this.createCard(
+      "Date Submitted: ",
+      moment(this.state.date, "x").format("LLLL")
```

Comment:
This all looks like a lot because VS-Code autoformatted everything, but this is the only line I actually changed